l10n_nl_tax_statement/views/l10n_nl_vat_statement_view.xml

```diff
@@ -0,0 +1,145 @@
+                            </group>
+                            <field name="line_ids" attrs="{'invisible': [('parent_id','!=',False)]}">
+                                <form>
+                                    <group>
+                                        <group>
+                                            <field name="code"/>
+                                            <field name="name"/>
+                                        </group>
+                                        <group>
+                                            <field name="omzet"/>
+                                            <field name="btw"/>
+                                        </group>
+                                    </group>
+                                </form>
+                                <tree decoration-muted="is_group" editable="bottom" create="false" delete="false">
+                                    <field name="is_group" invisible="1"/>
+                                    <field name="is_total" invisible="1"/>
+                                    <field name="is_readonly" invisible="1"/>
+                                    <field name="code" readonly="1"/>
+                                    <field name="name" readonly="1"/>
+                                    <field name="format_omzet" invisible="1"/>
+                                    <field name="format_btw" invisible="1"/>
+                                    <field name="omzet" attrs="{'invisible': [('format_omzet', '=', False)],'readonly': [('is_readonly', '=', True)]}"/>
+                                    <button name="view_base_lines" type="object" string="View base lines" icon="fa-search-plus" attrs="{'invisible': ['|',('format_omzet', '=', False),('is_total', '=', True)]}"/>
+                                    <field name="btw" attrs="{'invisible': [('format_btw', '=', False)],'readonly': [('is_readonly', '=', True)]}"/>
+                                    <button name="view_tax_lines" type="object" string="View tax lines" icon="fa-search-plus" attrs="{'invisible': ['|',('format_btw', '=', False),('is_total', '=', True)]}"/>
+                                </tree>
+                            </field>
+                            <group attrs="{'invisible': [('parent_id','!=',False)]}">
+                                <group>
+                                </group>
+                                <group>
+                                    <label for="btw_total"/>
+                                    <field name="btw_total" nolabel="1"/>
+                                </group>
+                            </group>
+                        </page>
+                        <page name="entry_lines" string="Past Undeclared Invoices" attrs="{'invisible':['|',('state','!=','draft'),('parent_id','!=',False)]}">
+                            <group name="unreported_move_filter" string="Include Undeclared Invoices">
```

Comment:
Add all undeclared invoices would be nice addition.

Reply:
Yes is a nice to have.

Reply:
Done in latest commit